src/app/component/eco-news/create-news/create-news.component.css

```diff
@@ -4,6 +4,7 @@
     height: 200px;
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/82373342-1df12880-9a26-11ea-9330-125fd3f75ff6.png)
Buttons should be aligned to the center(breakpoint 320px)
Also, please, resolve conflicts

Reply:
fixed